package.json

```diff
@@ -22,7 +22,9 @@
     "react-addons-css-transition-group": "^15.4.2",
     "react-bootstrap": "^0.30.0",
     "react-copy-to-clipboard": "^4.2.1",
+    "react-datepicker": "^0.44.0",
     "react-dropzone": "^3.11.0",
+    "react-fontawesome": "^1.5.0",
```

Comment:
It's in the actual component. It's not _super_ necessary, because it's only for the `<x>` in the top right, but I also feel like other components would use it too.